OpenRA.Mods.RA2/TraitsInterfaces.cs

```diff
@@ -0,0 +1,27 @@
+﻿#region Copyright & License Information
+/*
+ * Copyright 2015- OpenRA.Mods.AS Developers (see AUTHORS)
```

Comment:
And this is the AS header now, so we really need a header normalization within the project now.

Reply:
This should be changed to OpenRA's one for now, i guess.